packages/shared/src/graphql/posts.ts

```diff
@@ -362,16 +343,6 @@ export const dismissPostFeedback = (id: string): Promise<EmptyResponse> => {
   });
 };
 
-export const votePost = (
-  id: string,
-  vote: UserPostVote,
-): Promise<EmptyResponse> => {
-  return request(graphqlUrl, VOTE_POST_MUTATION, {
-    id,
-    vote,
-  });
-};
-
```

Comment:
Hmm this is not used but should we use it in `useVotePost` since it's the same thing?

Reply:
Based on requestMethod :)